tasks.py

```diff
@@ -43,15 +54,24 @@ def test(ctx, filter=None, junit=False):
         args += "--junit-xml test-results.xml"
     if filter is not None:
         args += " -k '{}'".format(filter)
-    ctx.run('python -m pytest {} {}'
-            .format(os.path.join(ROOT_DIR, "tests"), args))
+    ctx.run("python -m pytest {} {}".format(os.path.join(ROOT_DIR, "tests"), args))
+
+
+test_tasks.add_task(test, name="test", default=True)
+
+
+@task
+def release_test(ctx):
+    ctx.run("pip install .[test]")
+    test(ctx, junit=True)
+
 
-test_tasks.add_task(test, name='test', default=True)
+test_tasks.add_task(release_test, name="release")
```

Comment:
The name `release` for this task is somewhat misleading, as it doesn’t really perform a “release” of the package, it merely installs the package and runs the tests. While we don’t yet have a proper release script for the package, we will soon, and reserving this task name now will be an issue/conflict when we do.

Reply:
Do you have a suggestion for the task name?

Reply:
I would suggest adding a Boolean `install` option to the `test` task instead of adding a new task. That option, when set, would perform the install step before running the tests.